<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border:2px  solid #00007f;
padding:10px;
background-color: qlineargradient(spread:pad, x1:0.114697, y1:0.103, x2:1, y2:1, stop:0.447761 rgba(5, 70, 207, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="inputDate">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 92, 100);
border-radius:10px;
border-color: rgb(0, 0, 0);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
background-color: transparent;
</string>
    </property>
    <property name="text">
     <string>Fecha a consultar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 209, 122);
border-color: rgb(255, 255, 255);
border-radius:10px;
</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNombreUsuario">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>18</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 209, 122);
border:none;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelError">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(224, 27, 36);
border:none;
background:transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border:none;</string>
    </property>
    <property name="text">
     <string>Email a enviar la consulta</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>240</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 221, 218);
border:none;
color: rgb(36, 31, 49);
border-radius:14px;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
